### ISSUE TITLE
Hidden Layer Structure

### ISSUE THREAD

**Issue by PradyumnaSripad** [albertbup/deep-belief-network#28]
Hello sir,
This is a very good tf module i found for RBM and DBN, thanks for the same but I have a small doubt

In the below code snippet could you please tell me how to interpret 'hidden_layer_structure' parameter.

dbn = UnsupervisedDBN(hidden_layers_structure=[30,2],
                      batch_size=10,
                      learning_rate_rbm=0.06,
                      n_epochs_rbm=20,
                      activation_function='sigmoid')

**Comment by albertbup** [comment]
Hi,

In your case, hidden_layers_structure=[30,2] means you're creating a network with 2 hidden layers, the first one having 30 neurons and the latter having 2 neurons.
